### ISSUE TITLE
@angular/flex-layout@2.0.0-beta.10 requires a peer of @angular/core@0.0.0-NG but none is installed

### ISSUE THREAD

**Issue by akkumar** [angular/flex-layout#479]
#### Bug, feature request, or proposal:

Bug

#### What is the expected behavior?

cleaner npm install log

#### What is the current behavior?

Trying to install and save 2.0.0-beta.10 gives the following warnings. 

```console
$ npm install --save @angular/flex-layout@v2.0.0-beta.10
```
The warnings are below:

```console
npm WARN @angular/flex-layout@2.0.0-beta.10 requires a peer of @angular/core@0.0.0-NG but none is installed. You must install peer dependencies yourself.
npm WARN @angular/flex-layout@2.0.0-beta.10 requires a peer of @angular/common@0.0.0-NG but none is installed. You must install peer dependencies yourself.
```

#### What are the steps to reproduce?
```console
$ npm install --save @angular/flex-layout@v2.0.0-beta.10
```
gives the following warnings: 

```console
npm WARN @angular/flex-layout@2.0.0-beta.10 requires a peer of @angular/core@0.0.0-NG but none is installed. You must install peer dependencies yourself.
npm WARN @angular/flex-layout@2.0.0-beta.10 requires a peer of @angular/common@0.0.0-NG but none is installed. You must install peer dependencies yourself.
```
Stackblitz not applicable

#### What is the use-case or motivation for changing an existing behavior?

cleaner npm install log makes sure we don't overlook other warnings as applicable

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Angular: 4.4.6
Material:  2.0.0-beta.12
typescript: 2.4.2
OS: linux mint 
```console
$ lsb_release -a
No LSB modules are available.
Distributor ID:	LinuxMint
Description:	Linux Mint 18.2 Sonya
Release:	18.2
Codename:	sonya
```
browsers: not applicable.

```console
$ node --version
v8.4.0
```

```console
$ ng version

    _                      _                 ____ _     ___
   / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
  / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
 / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
/_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
               |___/
    
Angular CLI: 1.5.0
Node: 8.4.0
OS: linux x64
Angular: 
...
```

```console
$ npm version
{ npm: '5.5.1',
  ares: '1.10.1-DEV',
  cldr: '31.0.1',
  http_parser: '2.7.0',
  icu: '59.1',
  modules: '57',
  nghttp2: '1.22.0',
  node: '8.4.0',
  openssl: '1.0.2l',
  tz: '2017b',
  unicode: '9.0',
  uv: '1.13.1',
  v8: '6.0.286.52',
  zlib: '1.2.11' }
```

#### Is there anything else we should know?

This issue does not arise in beta.9 though . 
This seems to be ok. 
```bash
 npm install --save @angular/flex-layout@v2.0.0-beta.9 @angular/core@4.4.6 @angular/common@4.4.6 rxjs@5.5.0 zone.js@0.8.18
```
The following is not, though.
```bash
 npm install --save @angular/flex-layout@v2.0.0-beta.10 @angular/core@4.4.6 @angular/common@4.4.6 rxjs@5.5.0 zone.js@0.8.18
```

**Comment by avilao** [comment]
I have the same problem.

**Comment by ThomasBurleson** [comment]
This was a deployment bug. 
The release/deployment of v2.0.0-beta.10 has been fixed:

*  Peer dependencies updated.
*  ReadMe updated with **StackBlitz Templates**

**Comment by ThomasBurleson** [comment]
Reopened. We still need to re-deploy to NPM.

**Comment by pstephenwille** [comment]
ugh...`npm WARN @angular/flex-layout@2.0.0-rc.1 requires a peer of @angular/common@^2.2.3 but none is installed. You must install peer dependencies yourself.
`

**Comment by akkumar** [comment]
A new release seems to have been released recently that does not have this warning anymore. (it is oddly named , of course - And hopefully that is the inteded release for this fix, i presume).

```console
 npm install --save @angular/flex-layout@v2.0.0-beta.10-4905443  @angular/core@~4.4.4 @angular/common@~4.4.4 rxjs@^5.5.0 zone.js@^0.8.4
```
The above command gives a cleaner log.

**Comment by ThomasBurleson** [comment]
Fixed with today's 2.0.0-beta.12 release. See SHA 62457a5972 and [CHANGELOG](https://github.com/angular/flex-layout/blob/master/CHANGELOG.md)
